### DISCUSSION TITLE
What about zigbee2mqqt converter to ZHA handler?

### DISCUSSION THREAD

**Discussion by Vector-Int13** [zigpy/zha-device-handlers#1766] [Ideas]
I know it's a very bold idea, but why reinvent the wheel? 
If there is a tried and tested option for a given devicefor example ( _TZE200_lsanae15) for mqtt and there are a large number of such devices.
Is there anyone who can write a converter? That would make one integration into another.
I'm not that good at python... But then again, I have a lot of crazy ideas.

**Comment by Hedda** [comment]
Old question but FYI for reference, there were some similar questions and related discussions about porting Zigbee2MQTT converters here:

https://community.home-assistant.io/t/help-understanding-zha-quirks/411253/

https://github.com/zigpy/zigpy/discussions/934

PS: Also for reference, they are also actually not called Zigbee2MQTT or Z2M converter(s), but instead called zigbee-herdsman converters:

https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html

https://github.com/Koenkk/zigbee-herdsman-converters

**Comment by TheJulianJES** [comment]
It's not that simple. Quirks v2 should greatly improve quirks development speed though. It should also be easier to create new v2 quirks whilst referencing Z2M converters, especially for Tuya devices.

**Comment by mr-ransel** [comment]
I have a few devices lying around that I've been meaning to port, and it occurred to me that I _bet_ I could just dump them in an AI and get 90% of the way there, so I'll probably give that a go and figure out the right prompts and source code to pipe in.
